client/header/activity-panel/panels/orders.js

```diff
@@ -202,13 +202,21 @@ OrdersPanel.defaultProps = {
 export default compose(
 	withSelect( select => {
 		const { getReportItems, getReportItemsError, isReportItemsRequesting } = select( 'wc-api' );
+		const orderStatuses = wcSettings.wcAdminSettings.woocommerce_actionable_order_statuses || [
+			'processing',
+			'on-hold',
+		];
 		const ordersQuery = {
 			page: 1,
 			per_page: QUERY_DEFAULTS.pageSize,
-			status_is: [ 'processing', 'on-hold' ],
+			status_is: orderStatuses,
 			extended_info: true,
 		};
 
+		if ( ! orderStatuses.length ) {
```

Comment:
If this setting has no length, wondering if we should show a message in the panel along the lines of "Hey you currently have no order statuses set to display orders here!" Likely could be worded better, but otherwise the panel will always seem broken.

I do like the notion of bailing early before making the API request though 👍

Reply:
Good idea @timmyc.  Noted that issue in https://github.com/woocommerce/wc-admin/issues/1749 so we can follow-up on it.